### PR TITLE
To replace Func postfixed identifier with Fn postfixed in e2e extended

### PR DESCRIPTION
**Pull request by sdminonne**
@pmorie @smarterclayton built on top of https://github.com/openshift/origin/pull/5632
it sistematically replace `Func` with `Fn`.

Fixes https://github.com/openshift/origin/issues/5538

### CODE REVIEW

**Comment by sdminonne** [comment]
in extended e2e test only

**Comment by soltysh** [comment]
[testextendedonly][extended:core]

**Comment by soltysh** [comment]
Since this contains commit from #5632, can you just close the other one?

**Comment by soltysh** [comment]
LGTM

**Comment by sdminonne** [comment]
Not sure about tags added here https://github.com/openshift/origin/pull/5632#issuecomment-153376342 ... but I'm ok to close the other one.

**Comment by soltysh** [comment]
@sdminonne that tag is just for invoking tests, I've already did that.

**Comment by sdminonne** [comment]
Thanks. #5632 closed

**Comment by soltysh** [comment]
[test]

**Comment by soltysh** [comment]
This failed on testing assets, let me try again.
[test]

**Comment by soltysh** [comment]
This failed on testing assets, let me try again.
[extended:core]

**Comment by soltysh** [comment]
[testonlyextended][extended:core]

**Comment by soltysh** [comment]
The test are failing because of mysql replica test (https://github.com/openshift/origin/issues/5352), all the rest is ok.

**Comment by pmorie** [comment]
@sdminonne should be able to rebase this now

**Comment by sdminonne** [comment]
@pmorie oups I didn't see this thanks

**Comment by smarterclayton** [comment]
LGTM [merge] - thanks!

**Comment by soltysh** [comment]
The merge timed-out [here](https://github.com/openshift/origin/blob/08471b135e168e77de716fee9e888fe78cf5f249/test/end-to-end/core.sh#L275), with information:
```
Trying to pull repository docker.io/openshift/hello-openshift ... latest: Pulling from openshift/hello-openshift
...
1750b961071a: Layer already being pulled by another client. Waiting.
Timeout occurred while running ssh/sudo command: 
```

re-[merge]
